docs/examples/gen_velocities.rst

```diff
@@ -0,0 +1,432 @@
+*********************************************
+Generating synthetic scintillation velocities
+*********************************************
+
+This tutorial describes how to generate synthetic scintillation velocities for
+a pulsar in a circular orbit and a single one-dimensional screen, both with
+known parameters.
+
+Further explanations and derivations of the equations seen here can be found in
+Daniel Baker's "`Orbital Parameters and Distances
+<https://eor.cita.utoronto.ca/images/4/44/DB_Orbital_Parameters.pdf>`_"
+document. As in that document, the practical example here uses the parameter
+values for the pulsar PSR J0437--4715 as studied by `Reardon et al. (2020)
+<https://ui.adsabs.harvard.edu/abs/2020ApJ...904..104R/abstract>`_.
+
+Imports
+=======
+
+.. plot::
+    :include-source:
+    :context:
+    
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    from astropy import units as u
+    from astropy import constants as const
+
+    from astropy.time import Time
+    from astropy.coordinates import SkyCoord, SkyOffsetFrame, EarthLocation
+
+    from astropy.visualization import quantity_support, time_support
+
+Initializing parameters
+=======================
+
+Set the parameters of the pulsar system:
+
+.. list-table::
+    :widths: 2 1 2
+    :header-rows: 1
+
+    * - Parameter
+      - Symbol
+      - Remarks
+
+    * - **Coordinates of the pulsar system**
+      -  
+      -
+
+    * - declination
+      - :math:`\delta`
+      -
+
+    * - right ascension
+      - :math:`\alpha`
+      -
+
+    * - proper motion in declination
+      - :math:`\mu_\delta`
+      -
+
+    * - proper motion in right ascension
+      - :math:`\mu_{\alpha*}`
+      - including the :math:`\cos(\delta)` term
+
+    * - **Orbital elements of the pulsar binary**
+      -  
+      -
+    
+    * - binary period
+      - :math:`P_\mathrm{b}`
+      - 
+
+    * - projected semi-major axis
+      - :math:`a_\mathrm{p} \sin( i_\mathrm{p} )`
+      -
+
+    * - orbital inclination
+      - :math:`i_\mathrm{p}`
+      - between :math:`0^\circ` (face-on) and :math:`90^\circ` (edge-on)
```

Comment:
Is that (0 deg < i < 90 deg) correct, or can inclination be negative / up to 180 deg?

Reply:
Changed.